webapp/portlet/src/main/webapp/vue-apps/people-list/components/PeopleToolbar.vue

```diff
@@ -89,7 +71,20 @@ export default {
   data: () => ({
     filterToChange: null,
     bottomMenu: false,
+    startSearchAfterInMilliseconds: 300,
+    endTypingKeywordTimeout: 50,
+    startTypingKeywordTimeout: 0,
+    typing: false,
+    advancedFilterCount: 0,
+    showMobileFilter: false,
   }),
+  created() {
+    this.$root.$on('advanced-filter-count', (filterCount) => this.advancedFilterCount = filterCount );
+    this.$root.$on('reset-advanced-filter-count', () => {
+      this.advancedFilterCount = 0;
+      this.advancedFilterCountDisplay();
```

Comment:
Use verbs for methods

```suggestion
      this.updateAdvancedFilterCountDisplay();
```

Reply:
fixed